src/back/TicketManager/src/Core/TicketManager.Core.Services/Endpoints/Organizers/OrganizerListEndpoint.cs

```diff
@@ -0,0 +1,64 @@
+using System.Linq.Expressions;
+using FastEndpoints;
+using TicketManager.Core.Contracts.Common;
+using TicketManager.Core.Contracts.Organizers;
+using TicketManager.Core.Domain.Accounts;
+using TicketManager.Core.Domain.Organizer;
+using TicketManager.Core.Services.DataAccess;
+using TicketManager.Core.Services.DataAccess.DtoMappers;
+using TicketManager.Core.Services.Extensions;
+
+namespace TicketManager.Core.Services.Endpoints.Organizers;
+
+public class OrganizerListEndpoint: Endpoint<OrganizerListRequest, PaginatedResponse<OrganizerDto>>
+{
+    private readonly CoreDbContext coreDbContext;
+
+    public OrganizerListEndpoint(CoreDbContext coreDbContext)
+    {
+        this.coreDbContext = coreDbContext;
+    }
+
+    public override void Configure()
+    {
+        Get("/organizer/list");
+        Roles(AccountRoles.Admin);
+    }
+
+    public override async Task HandleAsync(OrganizerListRequest req, CancellationToken ct)
+    {
+        var query = coreDbContext.Organizers.AsQueryable();
+
+        if (req.CompanyNameFilter is not null)
+            query = query.Where(o => o.CompanyName.StartsWith(req.CompanyNameFilter));
+        if (req.AddressFilter is not null)
+            query = query.Where(o => o.Address.StartsWith(req.AddressFilter));
+        if (req.TaxIdFilter is not null)
+            query = query.Where(o => o.TaxId.StartsWith(req.TaxIdFilter));
+        if (req.TaxIdTypesFilter is not null)
```

Comment:
You can write `req.TaxIdFilter?.Any() ?? false` instead in case client send us empty list.

Reply:
I don't think we can receive empty list. This field is nullable so that is equivalent to empty list
We process lists in this form: `&TaxIdTypesFilter=0&TaxIdTypesFilter=1` not something like `&TaxIdTypesFilter=[0,1]` so empty list is probably not an option

Reply:
Actually it's not true. See how our list parser works.

Reply:
Actually you're right it's not possible.